static/app/components/charts/optionSelector.tsx

```diff
@@ -57,13 +57,13 @@ function OptionSelector({
 
   return (
     <CompactSelect
+      size="sm"
       options={mappedOptions}
       value={selected}
       onChange={onValueChange}
       isOptionDisabled={isOptionDisabled}
       multiple={multiple}
       triggerProps={{
-        size: 'sm',
```

Comment:
CompactSelect will forward the `size` prop to the button automatically (unless explicitly defined in `triggerProps`), so this is no longer necessary.